pkg/framework/aries/framework.go

```diff
@@ -39,6 +39,7 @@ type Aries struct {
 	transientStoreProvider storage.Provider
 	protocolSvcCreators    []api.ProtocolSvcCreator
 	services               []dispatcher.Service
+	msgHandler             dispatcher.MessageHandler
```

Comment:
This should not be a MessageHandler interface. The framework packages will not call Register and Unregister.

Reply:
changed to `MessageServiceProvider`. Hiding Register/Unregister from framework.